cartographer/mapping_2d/sparse_pose_graph.cc

```diff
@@ -614,7 +608,16 @@ transform::Rigid3d SparsePoseGraph::ComputeLocalToGlobalTransform(
   auto begin_it = submap_transforms.BeginOfTrajectory(trajectory_id);
   auto end_it = submap_transforms.EndOfTrajectory(trajectory_id);
   if (begin_it == end_it) {
-    return transform::Rigid3d::Identity();
+    if (initial_trajectory_poses_.find(trajectory_id) !=
+        initial_trajectory_poses_.end()) {
+      const InitialTrajectoryPose& initial_pose =
+          initial_trajectory_poses_.at(trajectory_id);
```

Comment:
Keep iterator of find to avoid doing find and at?

Reply:
Done. Thanks.